src/Context/alertContext.jsx

```diff
@@ -0,0 +1,21 @@
+import { createContext, useState } from "react";
+
+export const AlertContext = createContext({
+  showAlert: false,
+  alertMessage: null,
+  type: null,
+});
```

Comment:
True. Although, I would like to point out that when you set the properties here and say you're using useAlert custom hook somewhere else, on hovering it shows the properties that the context maintains which makes it easier to avoid bugs. Please find for reference:
1. Passing the object as  the default value as I have done
![image](https://user-images.githubusercontent.com/46674359/161364295-448266d9-882d-4dba-8fcf-f48e53524cb5.png)
2. Keeping it empty
![image](https://user-images.githubusercontent.com/46674359/161364321-5686ff2b-6b08-4ad4-a239-91fc2a198a10.png)

Reply:
Oh okay, that's great to know. Thanks for the info.